angular-spa-ui/js/common/commonService-srv.js

```diff
@@ -2,86 +2,75 @@
 
 module.exports = function (appLocalStorage, getTemplate) {
 
-    var initUserData = function (data, userData) {
-        userData['userId'] = data.data._id;
-        userData['userName'] = data.data.name;
-        userData['userEmail'] = data.data.email;
-        return userData;
-    };
+    return {
+        initUserData: function (data, userData) {
+            userData['userId'] = data.data._id;
+            userData['userName'] = data.data.name;
+            userData['userEmail'] = data.data.email;
+            return userData;
+        },
+        getToken: function (data, userData, tokenPath) {
+            var userTokenResponce = data.filter(function (responce) {
+                if (responce.url === tokenPath) {
+                    return responce;
+                }
+            });
 
-    var getToken = function (data, userData, tokenPath) {
-        var userTokenResponce = data.filter(function (responce) {
-            if (responce.url === tokenPath) {
-                return responce;
+            if (userTokenResponce.length === 1) {
```

Comment:
object.length is always integer, do not use strict comparing when you do not need it